pymatgen_diffusion/neb/tests/test_full_path_mapper.py

```diff
@@ -226,9 +242,23 @@ def test_integration(self):
         )
 
         self.cbg._tube_radius = 2
+        # self.cbg.populate_edges_with_chg_density_info()
+
+        # find this particular hop
+        ipos = [0.33079153, 0.18064031, 0.67945924]
+        epos = [0.33587514, -0.3461259, 1.15269302]
+        isite = PeriodicSite("Li", ipos, self.cbg.structure.lattice)
+        esite = PeriodicSite("Li", epos, self.cbg.structure.lattice)
+        ref_hop = MigrationHop(
+            isite=isite, esite=esite, symm_structure=self.cbg.symm_structure
+        )
+        hop_idx = -1
+        for k, d in self.cbg.unique_hops.items():
+            if d["hop"] == ref_hop:
+                hop_idx = k
```

Comment:
The change to pass symprec to MigrationHop can change the order in which unique IDs are assigned.
This fixes that problem by referencing the exact hop instead of just the index.